frameit/lib/frameit/editor.rb

```diff
@@ -56,7 +56,7 @@ def put_into_frame
       end
 
       @image = frame.composite(image, "png") do |c|
-        c.compose "Over"
+        c.compose "DstOver"
```

Comment:
Could you explain what this change does?

Reply:
Sure, before we were composing the screenshot “over” the frame, with the old devices it was fine since both the screenshot and the frame had a rectangular area. With iPhone X we have round corners and the notch, so we must have the screenshot on the lower level and the frame on top of it. To clarify, for old devices screenshot the result is exactly the same.

Reply:
Thanks for the reply @adellibovi! Just to confirm, you've tested it on old frames?

Reply:
Not all of them

Reply:
@ohayon Give me a second and I will upload some images from the example project, so you could see :)

Reply:
[en-US.zip](https://github.com/fastlane/fastlane/files/1434436/en-US.zip)
@ohayon have a look 😄

Reply:
Is it possible you could either confirm that all devices work or wrap that change in a check for the iPhone X? We are just trying to be super cautious to not break anyone's current workflow 😄

Reply:
@ohayon, checked and they all looks fine to me 👍